graderx/graders/moss/moss.py

```diff
@@ -29,7 +30,8 @@ def get_command(self):
 
         actual_files = []
         files = glob.glob(str(self.files_path.resolve()) + '/**/*', recursive = True) 
```

Comment:
I think this line could be simplified + use `sys.path` or `Path()` to join paths